src/scheduling/appointments/util/schedule-error-handling.util.ts

```diff
@@ -0,0 +1,28 @@
+import Appointment from 'model/Appointment'
```

Comment:
I don't like the naming of the file. Can you please remove `.util`since the file is already inside util folder?